schema/messaging.proto

```diff
@@ -358,7 +345,16 @@ message CreateMessageRequest {
   // The URI of the chat room or user profile that this message will be tied
   // to.
   string parent = 1 [
-    (google.api.resource_reference) = "google.showcase.v1alpha3.MessageParent",
+    (google.api.resource_set) = {
```

Comment:
@lukesneeringer, @chrisdunelm Which of these do you think makes more sense stylistically?

Reply:
Agreed. Fixed.